botocore/utils.py

```diff
@@ -3315,3 +3315,76 @@ def _serialize_if_needed(self, value, iso=False):
                 return value.isoformat()
             return value.strftime('%Y-%m-%dT%H:%M:%S%Z')
         return value
+
+
+SERVICE_NAME_ALIASES = {'runtime.sagemaker': 'sagemaker-runtime'}
+
+
+def handle_service_name_alias(service_name, **kwargs):
+    return SERVICE_NAME_ALIASES.get(service_name, service_name)
+
+
+# Mapping to determine the service ID for services that do not use it as the
+# model data directory name. The keys are the data directory name and the
+# values are the service IDs.
+CLIENT_NAME_TO_HYPHENIZED_SERVICE_ID_OVERRIDES = {
```

Comment:
To be clear, are you saying that the values are how customers would refer to the service in the config file and/or the endpoint prefix for the service? If so can you make that a bit more obvious here 😸

Reply:
This MUST be marked as non-public which I neglected to do!

I don't think customers ever need to refer to any of the values of this dictionary, they would only need the keys.

Reply:
maybe lets make it a private constant as well as indicating in a comment this is not intended for public use